indy_node/server/node.py

```diff
@@ -361,3 +370,29 @@ def is_request_need_quorum(self, msg_dict: dict):
         is_force = OPERATION in msg_dict and msg_dict.get(OPERATION).get(FORCE, False)
         is_force_upgrade = str(is_force) == 'True' and txn_type == POOL_UPGRADE
         return txn_type and not is_force_upgrade and super().is_request_need_quorum(msg_dict)
+
+    @staticmethod
+    def add_auth_rules_to_config_state(state: PruningState,
+                                       auth_map: dict,
+                                       serializer: AbstractConstraintSerializer):
+        for rule_id, auth_constraint in auth_map.items():
+            serialized_key = rule_id.encode()
+            serialized_value = serializer.serialize(auth_constraint)
+            if not state.get(serialized_key, isCommitted=False):
+                state.set(serialized_key, serialized_value)
+
+    def _init_write_request_validator(self):
+        constraint_serializer = ConstraintsSerializer(domain_state_serializer)
+        config_state = self.states[CONFIG_LEDGER_ID]
+        self.add_auth_rules_to_config_state(state=config_state,
```

Comment:
We should not modify state without consensus. If a rule is not present in the config state - use local auth map instead.